assets/prototype/domain/eventEditor/containers/mutations/useDeleteDateMutation.js

```diff
@@ -69,6 +71,17 @@ const useDeleteDatetimeMutation = ({ eventId, id }) => {
 					},
 				},
 			});
+			// Remove the datetime from all ticket relations
+			removeRelation({
+				entity: 'datetimes',
+				entityId: datetime.id,
+				relation: 'tickets',
+			});
```

Comment:
do we need to do the same for the event?
I would think that the event would have a relation set for the datetimes no?

Reply:
`eventId` gets passed along with the datetime mutation input

Reply:
ok... but do we need to do something like:

```js
removeRelation({
	entity: 'event',
	entityId: eventId,
	relation: 'datetimes',
	relationId: datetime.id,
});
```

Reply:
relations with the event are currently not maintained, the `eventId` is explicitly passed when creating a date. We don't need event relations for any other object, that's why I didn't add it.

Reply:
gotcha thnx 👍